dist/donut-chart/donut-chart.css

```diff
@@ -0,0 +1,18 @@
+.donut-chart {
+  align-items: center;
+  display: flex;
+  flex-wrap: wrap;
+  position: relative;
+}
+.donut-chart svg {
+  display: inline-block;
+  position: relative;
+  vertical-align: middle;
+}
+.donut-chart__segment {
+  opacity: 1;
+  transition: opacity 0.3s;
+}
+.donut-chart-inactive {
```

Comment:
Not sure of the use case here, but if this is something that toggles it might be better as a modifier (i.e. .donut-chart--inactive). Alternatively we may consider piggy backing off an ARIA attribute selector if any are suitable.

Reply:
good call was meant to be a modifier